sklearn/utils/estimator_checks.py

```diff
@@ -397,6 +398,39 @@ def check_dtype_object(name, Estimator):
 
 
 @ignore_warnings
+def check_dict(name, Estimator):
+    rnd = np.random.RandomState(0)
+    if name in ['SpectralCoclustering']:
+        return 0
+    if name in ['RANSACRegressor']:
+        X = 3 * rnd.uniform(size=(20, 3))
+    else:
+        X = 2 * rnd.uniform(size=(20, 3))
```

Comment:
If I use `3 * ` for all, I get `ValueError: _BinaryGaussianProcessClassifierLaplace supports only binary classification. y contains classes [0 1 2]`. Which is sounds fair enough.

With `2 *` for all I get on the other side `ValueError: No inliers found, possible cause is setting residual_threshold (None) too low.` for `RANSACRegressor ` :(